Castle/Classes/Util/CASUtils.m

```diff
@@ -37,9 +37,9 @@ void CASLog(NSString *format, ...)
     
     // Gather device information
     UIDevice *device = [UIDevice currentDevice];
-    NSString *deviceName = [device name];
+    NSString *model = [device name];
```

Comment:
won't this still get the device name instead of device model? Should it be `[device model]`?

Reply:
Yes it should